snapshot_dbg_cli/get_snapshot_command.py

```diff
@@ -84,8 +84,10 @@ def display_header(self, title):
 
   def display_summary(self, bp, status_message):
     location = breakpoint_utils.transform_location_to_file_line(bp['location'])
-    condition = bp.get('condition', 'No condition set.')
-    expressions = bp.get('expressions', 'No expressions set.')
+    condition = bp.get('condition', '')
+    condition = condition if condition != '' else 'No condition set'
+    expressions = bp.get('expressions', [])
+    expressions = expressions if len(expressions) > 0 else 'No expressions set'
```

Comment:
What are the cases where an empty string will be stored?

Reply:
I was just being defensive here to ensure the absence of the condition and expressions field is treated the same way as them being present but empty. As far as I know, there are no situations where we expect them to be present but empty as I don't believe the agents would populate them, and the CLI doesn't populate them if they are not specified by the user. I've added a comment to the code about use being defensive.